src/vma/util/lock_wrapper.h

```diff
@@ -68,7 +68,7 @@ class lock_base
 #endif
 protected:
 	tscval_t                start_lock_wait() { return 0; };
-	void                    end_lock_wait(tscval_t start_time) {start_time = 0; NOT_IN_USE(start_time);};
+	void                    end_lock_wait(tscval_t start_time) {start_time = 0; start_time = start_time;};
```

Comment:
1 .I can not explain. It is confused me too but using NOT_IN_USE() does not work when start_time=start_time does.
2. I did not investigate it.